vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,562 +1,225 @@
-
-## Implementing a New Engine or Integrating a New Task Source
-
-Implementing a new Engine for Process Driven Applications means to
-create a new Java Project and implement the interfaces defined in the
-pda-core-engine project. So, the new project should include the
-pda-core-engine as a dependency. To see an implementation in action,
-take a look at the pda-redhatpam-engine project, which implements the
-Red Hat PAM engine integration. After the engine is implemented, the JAR
-file resulting from the implementation should be available in the
-classpath for entando-process-driven-plugin, which is the project that
+value, and a timeframe selector. The Summary Card allows the
+application developer to select a request for rendering information. This request maps to a call in the underlying engine and provides the summarized data.
+
+![summary-cards.png](./pda-images/summary-cards.png)
+ 
+The PAM implementation of the Summary Card widget relies on the PAM custom query functionality. The PAM PDA engine exposes a configuration file where the custom query can be defined. This allows user customization of the data rendered on the summary cards. The application contains a "properties" file where the implementer can submit a custom query for each of the cards.
+ 
+#### Totals Over Time
+ 
+The Totals Over Time MFE provides a dual axis line/bar graph displaying
+trend information about the process environment. Three summary values can be compared over a single time period.
+
+![totals-over-time.png](./pda-images/totals-over-time.png)
+ 
+The PAM implementation of the Totals Over Time MFE utilizes custom queries to fetch the summary data rendered in the chart. The queries used in the implementation are defined in configuration files in the MFE and can be updated to render implementation specific data.
+ 
+### Process Automation Manager (PAM)
+ 
+The Entando PDA is built on Process Automation Manager, which is a business process automation engine built and maintained by Red Hat.
+ 
+### Backend for Frontend (BFF)
+ 
+A microservice architecture allows teams to iterate quickly and to
+develop technology to scale rapidly. Backend for Frontend
+is an architecture pattern built with microservices. The key
+component of this pattern is an application connecting the frontend
+of an application with the backend. The BFF Code Pattern helps to build that component according to IBM’s best practices.
+ 
+## Customization and Extension
+ 
+This page describes how the PDA plugin can be customized and extended via:
+
+- Styling
+- Implementing a new engine
+- Integrating a new task source
+
+For an in-depth discussion of the PDA architecture, refer to the [PDA Technical Guide](../customize-the-platform/pda-technical-guide.md).
+ 
+### Styling
+ 
+The Entando PDA MFEs are styled via a Material UI theme. That theme can
+be downloaded and updated [here](https://github.com/entando/frontend-libraries/tree/master/packages/entando-ui).
```

Comment:
This link is good. When you update the /next version with this content, you'll fix that error as well...